models/classes/media/LocalItemSource.php

```diff
@@ -179,6 +180,13 @@ public function delete($filename)
 
         return $deleted;
     }
+    
+    public function getFileStream($link)
+    {
+        $sysPath = $this->getSysPath($link);
+        $fh = fopen($sysPath, 'r');
+        return new Stream($fh, ['size' => filesize($sysPath)]);
```

Comment:
Why you need to pass the second parameter to the `Stream::_construct()` ?
There is used only one parameter:
https://github.com/slimphp/Slim/blob/3.x/Slim/Http/Stream.php#L82